### DISCUSSION TITLE
How are armor class values calculated for monsters in terms of ordering them?

### DISCUSSION THREAD

**Discussion by nathaniellee** [5e-bits/5e-srd-api#595] [Q&A]
I noticed that when ordering monsters by armor class, the Dryad, the Druid, the Archmage, and the Mage were all considered to have the highest values. They each have fairly low -- 11 or 12 -- "default" ACs and then some sort of bump for a second value.

It seems like in each of these cases, the total value of the two combined, however, is 27, which would put them above the highest solo AC which is the Tarrasque's 25.

However, there are three other monsters that also have multiple AC values: the Ankheg, the Azer, and the Lizardfolk. And in fact the Azer and the Lizardfolk's combined AC values would be even higher at 32 and 28, respectively.

The only thing I can suss out here is that perhaps it's correctly ignoring the second value when that value comes from a condition or armor, but combining the value when it comes from a spell?

**Comment by bagelbits** [comment]
I think you might be right.

**Comment by fergcb** [comment]
It seems to me that in all cases, the list of armor classes given are all alternatives. None of them are bonuses or meant to be summed up.

With spells like the Dryad's barkskin, they either have a DC of 11 if they haven't cast the spell, or 16 if they have. With armor like the Lizardfolk's shield, they get 13 while not wearing their shield, or 15 while wearing it. And with condition's like the Ankheg's `prone` condition, it gets an AC of 14 when standing and an AC of 11 while prone. Whereas the tarrasque just has a flat AC of 25 regardless of conditions, and it can't wear armor or cast any AC-modifying spells.

Are there any examples where this isn't the case, where we would be expected to sum the two values given to get the AC specified for the monster in the SRD?

**Reply by bagelbits** [reply]
@fergcb If you do a graphql query for monsters and order by AC descending, you'll see what is happening. This appears to be specific to the GraphQL endpoint.
